components/esp_matter_bridge/esp_matter_bridge.cpp

```diff
@@ -354,12 +325,18 @@ esp_err_t remove_device(device_t *bridged_device)
     return error;
 }
 
-esp_err_t initialize(node_t *node)
+esp_err_t initialize(node_t *node, bridge_device_type_callback_t device_type_cb)
 {
     if (!node) {
         ESP_LOGE(TAG, "node could not be NULL");
         return ESP_ERR_INVALID_ARG;
     }
+    
+    if (!device_type_cb) {
+        ESP_LOGE(TAG, "device_type_callback cannot be NULL, use set_device_type_cb");
```

Comment:
Is `set_device_type_cb` defined somewhere?
```suggestion
        ESP_LOGE(TAG, "device_type_callback cannot be NULL.");
```